skbuild/setuptools_wrap.py

```diff
@@ -59,19 +63,21 @@ def parse_args():
                              concatenate_value=True)
     dutils, cmake = move_arg('-G', dutils, cmake)
     dutils, make = move_arg('-j', dutils, make)
-    op = os.path
-
-    def absappend(x):
-        return op.join(op.dirname(op.abspath(sys.argv[0])), x)
+    absappend = (
+        lambda x: os.path.join(
```

Comment:
JC provided a link in the patch linked:
http://stackoverflow.com/questions/25010167/e731-do-not-assign-a-lambda-expression-use-a-def

PEP-8